SYCL/ESIMD/lsc/Inputs/lsc_store_2d.hpp

```diff
@@ -16,7 +16,38 @@
 using namespace sycl;
 using namespace sycl::ext::intel::esimd;
 using namespace sycl::ext::intel::experimental::esimd;
-using namespace sycl::ext::intel::experimental::esimd::detail;
+
+template <unsigned int N, unsigned int M>
```

Comment:
These functions below were added to avoid usage of experimental::esimd::detail utilities in E2E tests.